File: code-coaches/medium/hex-color-code-generator/HexColorCodeGenerator.cs
using System;

public class HexColorCodeGenerator
{

    static string RgbToHex(int r, int g, int b)
    {
        // Convert each RGB value to a two-digit hex string
        string hexCode = String.Format("#{0:x2}{1:x2}{2:x2}", r, g, b);
        return hexCode;
    }

    static void Main(string[] args)
    {
        int rRgb = int.Parse(Console.ReadLine());
        int gRgb = int.Parse(Console.ReadLine());
        int bRgb = int.Parse(Console.ReadLine());

        string hexCode = RgbToHex(rRgb, gRgb, bRgb);
        Console.WriteLine(hexCode);
    }
}

File: code-coaches/medium/hex-color-code-generator/HexColorCodeGenerator.swift
import Foundation

func rgbToHex(r: Int, g: Int, b: Int) -> String {
    // Convert each RGB value to a two-digit hex string
    let hexCode = String(stringInterpolation: "#%02x%02x%02x", r, g, b)
    return hexCode
}

func main() {
    let rRgb = Int(readLine()!)!
    let gRgb = Int(readLine()!)!
    let bRgb = Int(readLine()!)!
	
	print(rgbToHex(r: rRgb, g: gRgb, b: bRgb))
}

main()